test/extended/networking/network_segmentation.go

```diff
@@ -212,15 +212,8 @@ var _ = Describe("[sig-network][OCPFeatureGate:NetworkSegmentation][Feature:User
 						f.Namespace = ns
 						By("Creating second namespace for default network pods")
 						defaultNetNamespace := f.Namespace.Name + "-default"
-						_, err = cs.CoreV1().Namespaces().Create(context.Background(), &v1.Namespace{
-							ObjectMeta: metav1.ObjectMeta{
-								Name: defaultNetNamespace,
-							},
-						}, metav1.CreateOptions{})
+						_, err = f.CreateNamespace(context.TODO(), defaultNetNamespace, nil)
```

Comment:
question @tssurya: don't we need to `err = udnWaitForOpenShift(oc, namespace)` here ?

Reply:
because that namespaces isn't a UDN namespace... unsure if that `udnWaitForOCP` is doing generic things or not.
but it does make to add this wait also for the non UDN namespaces huh?

Reply:
I'm unsure ...